### ISSUE TITLE
Phonetische Suche nach Kind

### ISSUE THREAD

**Issue by wintermeyer** [wort-schule/wort.schule#327]
WIP

https://wort.schule/?filterrific%5Bfilter_home%5D=kend

<img width="685" alt="Bildschirm­foto 2023-04-24 um 15 59 31" src="https://user-images.githubusercontent.com/69770/234019978-d78b5779-3f3c-42c6-bc6b-8a2d05d3f27a.png">

https://wort.schule/?filterrific%5Bfilter_home%5D=kint

<img width="686" alt="Bildschirm­foto 2023-04-24 um 16 01 41" src="https://user-images.githubusercontent.com/69770/234019847-9a42c0d7-c25f-4f5a-9ffc-b186f03f8c4a.png">

**Comment by wintermeyer** [comment]
@phylor Kannst Du mal überprüfen, ob der Cologne Algorithmus korrekt eingebaut ist? Ich will nur auf Nummer sicher gehen, weil hier ein paar merkwürdige Treffer drin sind (ich habe den Algorithmus aber nicht im Kopf).

**Comment by wintermeyer** [comment]
Ich überlege gerade, ob wir bessere Treffer bekommen, wenn wir die „Entfernen aller Codes „0“ außer am Anfang“ Regel nicht befolgen. 

https://de.wikipedia.org/wiki/Kölner_Phonetik

**Comment by wintermeyer** [comment]
Je länger ich mir das alles anschaue, desto mehr komme ich zum Schluss, das Kölner Phonetics für uns zu unscharf ist. Da sind viel zu viele false Positives drin. Das wurde auch zu einem anderen Zweck geschrieben und auch in einer ganz anderen Zeit (viel weniger Computer Power). Die Alternativen sind auch alle nichts für uns. Entweder wir biegen Kölner Phonetics für unsere Zwecke zurecht oder wir erstellen einen neuen Algorithmus.

**Comment by wintermeyer** [comment]
> @phylor Kannst Du mal überprüfen, ob der Cologne Algorithmus korrekt eingebaut ist? Ich will nur auf Nummer sicher gehen, weil hier ein paar merkwürdige Treffer drin sind (ich habe den Algorithmus aber nicht im Kopf).

@phylor Vergiss es bitte. Stimmt alles.

**Comment by cabotine** [comment]
Ja, deswegen hatte ich überlegt, ob man es mit dem Trigram Ansatz kombiniert.

**Comment by cabotine** [comment]
Die phonetische Suche berücksichtigt noch nicht die anderen Wortformen oder?
Kender führt damit noch nicht zu Kinder
![grafik](https://user-images.githubusercontent.com/8180537/234266134-2b000ba6-20e9-4dc5-9723-ccb855f93d6e.png)

**Comment by phylor** [comment]
> Die phonetische Suche berücksichtigt noch nicht die anderen Wortformen oder? Kender führt damit noch nicht zu Kinder

Korrekt, die phonetische Suche sucht nur in der Grundform.

**Comment by phylor** [comment]
@wintermeyer Welche Formen des Verbes verwenden wir für die phonetische Suche? Nur die Grundform? Oder auch andere Formen? Hier die Auswahl:

```
      participle: "rennend",
      past_participle: "gerannt",
      present_singular_1: "renne",
      present_singular_2: "rennst",
      present_singular_3: "rennt",
      present_plural_1: "rennen",
      present_plural_2: "rennt",
      present_plural_3: "rennen",
      past_singular_1: "rannte",
      past_singular_2: "ranntest",
      past_singular_3: "rannten",
      past_plural_1: "rannten",
      past_plural_2: "ranntet",
      past_plural_3: "rannten",
```

**Comment by wintermeyer** [comment]
> @wintermeyer Welche Formen des Verbes verwenden wir für die phonetische Suche?

Alle. Bis zu dem Tag, wo wir merken das das eine blöde Idee war.
